src/boot/api.ts

```diff
@@ -38,11 +39,11 @@ export default boot(async ({ store }) => {
 
   const networkIdx = computed(() => store.getters['general/networkIdx']);
 
-  const defaultEndpoint = providerEndpoints[networkIdx.value].endpoints[0].endpoint;
+  const randomEndpoint = getRandomFromArray(providerEndpoints[networkIdx.value].endpoints).endpoint;
```

Comment:
https://user-images.githubusercontent.com/40356749/166207222-b158509a-2277-4727-9970-ec6973f4c68f.mov

When the user switches the network, the endpoint should be randomized, but when I try to switch from one network to another, the modal will revert my option back to the first option.

Reply:
Fixed.

Demo:
https://www.loom.com/share/ed6a8232ea9f465dab28f9facd3cdfb0

Code:
https://github.com/AstarNetwork/astar-apps/pull/375/commits/15eaff184ed6494592c50a796f063fcafb5d8466
https://github.com/AstarNetwork/astar-apps/pull/375/commits/0560a87967d0baafb0f6cea7d94bbdf5033276dd